woocommerce/woocommerce-ios
# Order Details: present shipping label creation info screen on info link row tap

Fixes #2975 

- [x] ⚠️ Make sure PR base is `develop` after https://github.com/woocommerce/woocommerce-ios/pull/3769 is merged ⚠️ 

## Why

This PR implemented the shipping label creation info screen when the user taps on the info link row in order details. The info screen includes a CTA that opens a webview.

## Changes

- Added image assets for light and dark mode named `woo-shipping-label-creation`
- Created `ShippingLabelCreationInfoViewController` that embeds `EmptyStateViewController` as a child view controller with "Learn more" CTA that opens up a link

## Testing

- Go to the orders tab
- Tap an order
- Tap "Learn more about creating labels with your phone" in order details --> should see an info screen with a "Learn more" link that presents a webview with more details

## Example screenshots

Screencast:

https://user-images.githubusercontent.com/1945542/109902717-cb8fa880-7cd5-11eb-8e57-73e4ce019377.mov

dark | light
-- | --
![Simulator Screen Shot - iPhone 11 - 2021-03-04 at 10 28 49](https://user-images.githubusercontent.com/1945542/109902698-c5013100-7cd5-11eb-82e5-4988e1112be3.png) | ![Simulator Screen Shot - iPhone 11 - 2021-03-04 at 10 29 03](https://user-images.githubusercontent.com/1945542/109902712-c9c5e500-7cd5-11eb-8d85-e38cf7be71d8.png)




Update release notes:
